core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/convert.kt

```diff
@@ -402,17 +482,51 @@ public fun DataColumn<String?>.convertToLocalTime(
     return map { it?.let { converter(it.trim()) ?: error("Can't convert `$it` to LocalTime") } }
 }
 
+@JvmName("toLocalTimeFromTLongNullable")
+@Refine
+@Converter(LocalTime::class, nullable = true)
```

Comment:
It's kind of pattern to define type info for outputs, isn't it? a-la @ReturnType as a more common approach